scipy/ndimage/_morphology.py

```diff
@@ -202,6 +202,24 @@ def generate_binary_structure(rank, connectivity):
             [False,  True, False],
             [False, False, False]]], dtype=bool)
 
+    >>> import matplotlib.pyplot as plt
+    >>> import scipy.ndimage
```

Comment:
```suggestion
```

Reply:
But then, why should this go in the documentation although the other examples we added went into the tutorial? It seems like this could go above your addition of `morphology_binary_dilation_erosion`.

Reply:
Good point!
Later today, I will add it there.
Also, I have the feeling that some of these big examples are in their current form not the perfect match for this kind of sphinx documentation. Most of the time you need only one line of code, and you don't bother too much about the imports or the array definitions. Then, instead of just copy and pasting this line of code, one has to search the whole example for it. Also, it clutters the documentation.  And otherwise, when not showing the code at all, the line one wants to see is missing as well. Here I just made a new prototype that could be a nice middle course: https://kolibril13.github.io/plywood/
As this is only a prototype and I will iterate a lot over it in the next few days, I only mention it right now here in this latent place.